src/Services/Asset/AssetManager.php

```diff
@@ -70,10 +70,13 @@ public function core()
     /**
      * @param array $customAssets
      * @return string
+     * @throws \Throwable
      */
-    public function bundle(array $customAssets = [])
+    public function bundle(array $customAssets = []): string
     {
         if ($this->js) {
+            $customAssets = array_merge($customAssets, config('laradium.custom_js', []));
```

Comment:
This should be moved to a separate public method. In order to be able to get customAssets without bundle assets.